### PR TITLE
[psmisc]:  Add inspec tests

### PR DESCRIPTION
**Pull request by gavindidrichsen**

### CODE REVIEW

File: controls/psmisc_exists.rb [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    it { should exist }
+    it { should be_executable }
+  end
+
+  # pstree
+  pstree_full_path = File.join(plan_installation_directory.stdout.strip, "bin/pstree")
+  describe file(pstree_full_path) do
+    it { should exist }
+    it { should be_executable }
+  end
```
Comment (MindNumbing) [inline controls/psmisc_exists.rb]: May be best to convert these into a looped existance check, see [here](https://github.com/chef-base-plans/binutils/blob/177b5d4f4f4a262cdf05e149638e27b14f5a3056/controls/binary_tests.rb#L25-L31) for an example.

File: controls/psmisc_works.rb [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  end
+
+  # pstree
+  pstree_full_path = File.join(plan_installation_directory.stdout.strip, "bin/pstree")
+  describe command("#{pstree_full_path} --version") do
+    its('exit_status') { should eq 0 }
+    its('stderr') { should_not be_empty }
+    its('stderr') { should match /pstree \(PSmisc\) #{plan_pkg_version}/ }
+    its('stdout') { should be_empty }
+  end
```
Comment (MindNumbing) [inline controls/psmisc_works.rb]: These could also be looped, see [here](https://github.com/chef-base-plans/binutils/blob/177b5d4f4f4a262cdf05e149638e27b14f5a3056/controls/binary_tests.rb#L35-L40) for an example.

**Review by MindNumbing — CHANGES_REQUESTED** [review]
A couple of control changes.

File: controls/psmisc_exists.rb (hunk 1 above)
Comment (gavindidrichsen) [reply controls/psmisc_exists.rb]: @MindNumbing:  converted to array loop

File: controls/psmisc_works.rb (hunk 2 above)
Comment (gavindidrichsen) [reply controls/psmisc_works.rb]: Converted to array loop

**Review by MindNumbing — APPROVED** [review]
Thanks @gavindidrichsen, LGTM! 👍

**Pull request merged**